src/main/java/sirius/db/es/BulkContext.java

```diff
@@ -104,21 +120,28 @@ public BulkContext forceDelete(ElasticEntity entity) {
     }
 
     private void update(ElasticEntity entity, boolean force) {
+        isUpdateRequest = true;
```

Comment:
deletes and updates can be mixed - shouldn't we really rely on this. I'd rather not invoke any afterXXX calls (and document this behaviour) - a bulk delete with afterDeleteCalls wouldn`t be much of a performance win anyway..

Also storing all entities in another list might add some heap pressure (unexpected)